lib/main.dart

```diff
@@ -1,78 +1,110 @@
-import 'package:firebase_auth/firebase_auth.dart';
 import 'package:firebase_core/firebase_core.dart';
 import 'package:flutter/material.dart';
-import 'package:flutterfire_ui/auth.dart';
+import 'package:go_router/go_router.dart';
+import 'package:meeting_scheduler/auth/auth_gate.dart';
+import 'package:meeting_scheduler/meeting/screen_meeting_details.dart';
+import 'package:meeting_scheduler/not-found/not_found_screen.dart';
+import 'package:meeting_scheduler/organization/screen_org_details.dart';
+import 'package:meeting_scheduler/profile/screen_profile_details.dart';
+import 'package:meeting_scheduler/project/screen_project_details.dart';
+import 'package:meeting_scheduler/team/screen_team_details.dart';
 
 import 'firebase_options.dart';
 
 void main() {
-  runApp(const MyApp());
+  runApp(MyApp());
 }
 
 class MyApp extends StatelessWidget {
-  const MyApp({Key? key}) : super(key: key);
```

Comment:
Why did you remove const here?

Reply:
@OppositeDragon did you see this question?

Reply:
I imagine there was a non-const member added?  I just couldn't tell from a quick look and thought I'd be lazy and ask 🙂